src/server/templates/typescript.ts

```diff
@@ -419,7 +435,15 @@ const pgTypeToTsType = (
       }
       return 'unknown'
     }
-
+    const tableRowType = tables.find((table) => table.name === pgType)
+    if (tableRowType) {
+      if (schemas.some(({ name }) => name === tableRowType.schema)) {
+        return `Database[${JSON.stringify(tableRowType.schema)}]['Tables'][${JSON.stringify(
+          tableRowType.name
+        )}]['Row']`
+      }
+      return 'unknown'
+    }
```

Comment:
If the type is yet to be found we search in the tables of the same schema. If we find it we return the row type for that table.